pkg/cloudprovider/kubevirt/cloud.go

```diff
@@ -20,11 +17,8 @@ const (
 	ProviderName = "kubevirt"
 )
 
-var scheme = runtime.NewScheme()
-
 func init() {
-	corev1.AddToScheme(scheme)
```

Comment:
Don't we need the client schemes to construct a proper client later?
I think the initialisation of the scheme needs to stay here.

Reply:
KubeVirt already registers appropriate schema here:
https://github.com/kubevirt/kubevirt/blob/main/staging/src/kubevirt.io/client-go/api/v1/register.go#L93

Reply:
Ok, that's cool! But what ab

Reply:
The same happens for `corev1` https://github.com/kubernetes/api/blob/master/core/v1/register.go, basically each API should register itself that we don't have to do it ourselves before client creation.

Reply:
You are right! really nice :)